markets/perps-market/test/integration/bootstrap/bootstrapTraders.ts

```diff
@@ -0,0 +1,68 @@
+import { bootstrapStakers } from '@synthetixio/main/test/common';
+import { Systems } from './bootstrap';
+import { snapshotCheckpoint } from '@synthetixio/core-utils/utils/mocha/snapshot';
+import { ethers } from 'ethers';
+
+type Data = {
+  systems: () => Systems;
+  signers: () => ethers.Signer[];
+  provider: () => ethers.providers.JsonRpcProvider;
+  accountIds: Array<number>;
+};
+/*
+  creates two traders with the specified account ids.
+  a potential enhancement is maybe to allow x # of traders but i don't expect more than 2 traders
+  needed for testing
+*/
+export function bootstrapTraders(data: Data) {
+  const { systems, signers, provider, accountIds } = data;
+  bootstrapStakers(systems, signers);
+
+  let trader1: ethers.Signer, trader2: ethers.Signer;
+
+  before('provide access to create account', async () => {
+    const [owner, , , trader1, trader2] = signers();
```

Comment:
this pattern might hide trader1 and trader2 from definition 3 lines above. Won't be a problem in this `before` but if used in another place can hide a difficult to fix issue

Reply:
well, it returns `trader1` and `trader2` as top level keys in this function so in a test, you would use those functions instead of using the 4th and 5th index in `signers`.  similar pattern to how we source `owner` and `marketOwner`